examples/trianglell/main.rs

```diff
@@ -308,6 +308,28 @@ fn main() {
         // present frame
         swap_chain.present();
     }
+
+    // cleanup!
+    factory.destroy_shader_lib(shader_lib);
+    factory.destroy_pipeline_layout(pipeline_layout);
+    factory.destroy_renderpass(render_pass);
+    factory.destroy_heap(heap);
+    factory.destroy_heap(image_heap);
+    factory.destroy_heap(image_upload_heap);
+    factory.destroy_buffer(vertex_buffer);
+    factory.destroy_buffer(image_upload_buffer);
+    factory.destroy_image(image_logo);
+    for pipeline in pipelines.into_iter() {
```

Comment:
Should be required as we need ownership of the pipeline and `iter()/iter_mut()` iterate only over references.
Couldn't make the compiler happy without using `into_iter()` either.